.gitignore

```diff
@@ -14,3 +14,4 @@ docs/reference/src/main/docbook/en-US/version_info.xml
 .forge_settings
 .factorypath
 atlassian-ide-plugin.xml
+/*/bin/
```

Comment:
In fact you shouldn't have a bin folder if you're compiling using Maven. On what situations do you see a bin folder in the project root?

Reply:
It was auto-genercate by eclipse, I guess